### DISCUSSION TITLE
Filter cell barcodes returns to differnet cell number

### DISCUSSION THREAD

**Discussion by shuangcao-mhh** [aertslab/scenicplus#212] [Q&A] [answered]
Hi, I am trying the scenicplus pipline with the example pbmc data. But after the step of barcode filtration, then there is very less cell remained. I used exactly the same codes as the tutorial. 

![image](https://github.com/aertslab/scenicplus/assets/89731915/ac485213-a3ac-47a0-8250-97894ff57cbb)

UserWarning: 

`distplot` is a deprecated function and will be removed in seaborn v0.14.0.

Please adapt your code to use either `displot` (a figure-level function with
similar flexibility) or `histplot` (an axes-level function for histograms).

For a guide to updating your code to use the new functions, please see
https://gist.github.com/mwaskom/de44147ed2974457ad6372750bbe5751

Does anyone have any idea what's wrong? 
Thanks in advance!

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @Annie133 

this seems similar to this bug https://github.com/aertslab/scenicplus/issues/146#issuecomment-1518740127

What is your pandas version?

Best,

Seppe

**Reply by shuangcao-mhh** [reply]
Hi, 

I checked my pandas version, it is 1.5.0. But still has the bug, 0 barcodes passed QC stats. 
![image](https://github.com/aertslab/scenicplus/assets/89731915/c57fe75b-5991-446a-98f0-45050cf48a81)

Best,
Shuang

**Reply by SeppeDeWinter** [reply]
Hi,

With 

```python

import pandas as pd
pd.__version__

```

you also see version 1.5.0?

Best,

Seppe

**Reply by shuangcao-mhh** [reply]
Yes, 
![image](https://github.com/aertslab/scenicplus/assets/89731915/25251853-e901-488b-90bf-45c6008b5016)

I am wondering whether it is caused by other conflicts. 

Solve it! By re-install.  Thank you so much for the kind help!
Best,
Shuang
